lib/howl/interactions/file_selection.moon

```diff
@@ -108,21 +112,20 @@ class FileSelector
   keymap:
     enter: =>
       app.editor\cancel_preview!
+      file = @list_widget.selection and @list_widget.selection.file
       name = @list_widget.selection and @list_widget.selection.name
-      if not @allow_new and not name
-        log.error 'Invalid path'
+      if not @opts.allow_new and (not file or not file.exists)
+        log.error 'Invalid path: '..tostring(file)
```

Comment:
Both works, but I thought I'd point out that this could also be the shorter `"Invalid path: #{file}".